### PR TITLE
Some French translations + Fixed spaces

### PR DESCRIPTION
**Pull request by Tititesouris**
I'm not sure if the double space was a mistake or not, and I need to take a break for today so I'll finish tomorrow with the translations

### CODE REVIEW

**Comment by cedeber** [comment]
The double spaces are in sentences generated from Django views. I guess we need to keep the mistake on the English sentences a least so that the translation can still work.
I will dive into it to see if I can found any info about it.

File: locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -202,71 +202,79 @@ msgid ""
...
 "Forgotten your password? Enter your email address below, and we'll email "
 "instructions for setting a new one."
 msgstr ""
+"Mot de passe oublié ? Entrez votre email ci-dessous et vous recevrez "
+"les instructions pour avoir un nouveau mot de passe."
 
 #: member/templates/member/password_reset.html:17
 msgid "Email address:"
-msgstr ""
+msgstr "Adresse email"
```
Comment (cedeber) [inline locale/fr/LC_MESSAGES/django.po]: Je pense qu'ici adresse électronique est mieux, question de constance. Et il manque le `:`

**Comment by Tititesouris** [comment]
It should be all good now. I've used "adresse électronique" when speaking about the email adresse and "email" when speaking about the protocol (send via email/envoyer par email)

**Comment by Tititesouris** [comment]
Should be good now, thanks for checking!

**Comment by cedeber** [comment]
Looks good. Merci.

**Pull request merged**